### PR TITLE
Use Fudge for Windows development and CI

### PR DESCRIPTION
**Pull request by jayvdb**
Includes workaround for https://github.com/coala/coala/issues/5985

Note commit `.moban.yaml: Use local mobans` needs to be removed before merging.

### CODE REVIEW

File: .misc/appveyor.yml [hunk 1]
```diff
@@ -32,32 +41,90 @@ branches:
...
 
 test_script:
-  - "%CMD_IN_ENV% py.test"
-  - "%CMD_IN_ENV% python setup.py install"
-  - "%CMD_IN_ENV% python -m pip install \
-    git+https://github.com/coala/coala-bears"
+  - python -m pip --version
+  - python -c "import setuptools; print(setuptools.__version__)"
+
+  - py.test
```
Comment (abhishalya) [inline .misc/appveyor.yml]: Isn't `py.test` old enough to be replaced by `pytest`?

File: .misc/appveyor.yml [hunk 2]
```diff
@@ -32,32 +41,90 @@ branches:
...
+  - python -m pip install
+      git+https://github.com/coala/coala-bears#egg=coala-bears
   - npm install csslint -g
   # https://github.com/coala/coala/issues/5619
   - rm docs/Developers/coala_settings.rst
-  - sed -i '/ShellCheckBear/d' .coafile
-  - "%CMD_IN_ENV% coala --ci"
+  # https://github.com/coala/coala/issues/5985 and
+  # https://github.com/coala/coala/issues/6028 and others:
+  - rm coalib/misc/Asyncio.py
```
Comment (abhishalya) [inline .misc/appveyor.yml]: Oh, I thought you had a different hacky fix :P

File: .misc/Fudge.ps1 [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+                Write-Fail "Action not recognised: $($_)"
+            }
+    }
+}
+finally
+{
+    # output duration, and cleanup
+    Write-Details "`nDuration: $(([DateTime]::UtcNow - $timer).ToString())"
+    Remove-Module -Name 'FudgeTools' -ErrorAction SilentlyContinue | Out-Null
+}
```
Comment (abhishalya) [inline .misc/Fudge.ps1]: EOL at EOF?

File: .misc/appveyor.yml (hunk 1 above)
Comment (jayvdb) [reply .misc/appveyor.yml]: Sure fix it in the template

File: .misc/Fudge.ps1 (hunk 3 above)
Comment (jayvdb) [reply .misc/Fudge.ps1]: this is code imported from https://github.com/Badgerati/Fudge ; raise issues or PRs there :P

File: .misc/appveyor.yml (hunk 2 above)
Comment (jayvdb) [reply .misc/appveyor.yml]: Nope - a hack until we get a real solution, or drop py34 (but py34 is helpful for the bears travis trusty (and precise?) builds, which are needed for some of the languages)

**Pull request merged**